extra/linux/dev.lapce.lapce.desktop

```diff
@@ -5,7 +5,15 @@ Type=Application
 Name=Lapce
 Comment=Lightning-fast and powerful code editor written in Rust
 Categories=Development;IDE;
+GenericName=Code Editor
 
 Icon=dev.lapce.lapce
 Exec=lapce
 Terminal=false
+MimeType=text/plain;inode/directory;
+Actions=new-empty-window;
+
+[Desktop Action new-empty-window]
+Name=New Empty Window
+Exec=lapce --new %F
```

Comment:
That doesn't seem to be "empty"?

Reply:
It depends on the context the action is used in, but i guess just "new window" would be better (I based this on the vscode desktop files)